18 Normalizr/src/pages/student-detail/student.validation.ts

```diff
@@ -26,6 +28,11 @@ class StudentFormValidation extends BaseFormValidation {
       "email",
       emailValidationHandler,
     );
+
+    this._validationEngine.addFieldValidation(
+      "country",
+      requiredIdValidationHandler,
```

Comment:
Thanks, we have to review and evolve it. Some weeks ago we added chain patter to addFieldValidation, we plan as well to add an optional field to add more context to some scenarios (e.g. maxlenght validation in that extra param you would indicate the max lenght number, or match to fields, in that extra param you can add the field to compare against).

Another idea that we have in mind is enable users to define the validation to apply in a JSON file.